stdlib/public/runtime/Errors.cpp

```diff
@@ -140,16 +141,16 @@ void swift::dumpStackTraceEntry(unsigned index, void *framePC,
     fprintf(stderr, "%s`%s + %td", libraryName.data(), symbolName.c_str(),
             offset);
   } else {
-    constexpr const char *format = "%-4u %-34s 0x%0.16lx %s + %td\n";
-    fprintf(stderr, format, index, libraryName.data(), symbolAddr,
-            symbolName.c_str(), offset);
+    constexpr const char *format = "%-4u %-34s 0x%0.16tx %s + %td\n";
+    fprintf(stderr, format, index, libraryName.data(),
+            static_cast<uintptr_t>(symbolAddr), symbolName.c_str(), offset);
```

Comment:
`symbolAddr` is already a `uintptr_t`.

Reply:
GCC 6 seemed to emit conversion warnings for it even with `%tx`.  I think it was a `void *`.

Reply:
`framePC` is a `void *`, but `symbolAddr` isn't. Copy/paste from below, maybe?

Reply:
I'm going to go with, I'm just losing my mind.  Verified that it isn't an issue.  Thanks for catching that!